tox.ini

```
@@ -1,35 +1,34 @@
 [tox]
 envlist =
-    py{36,37,38,39}-django{22,30,31,master}
+    py{36,37,38,39}-dj{21,22,30,31,master}
```

I think we dropped support for 2.1

```suggestion
    py{36,37,38,39}-dj{22,30,31,master}
```

---

Good catch!